src/istgt_iscsi.c

```diff
@@ -5724,7 +5745,7 @@ sender(void *arg)
 	pthread_set_name_np(slf, tinfo);
 #endif
 
-	pthread_cleanup_push(snd_cleanup, (void *)conn);
+//	pthread_cleanup_push(snd_cleanup, (void *)conn);
```

Comment:
This seems not needed for the change introduced as we are going to dump core anyway.

Reply:
This change is not related to this issue. But, as the other code is removed which takes care of unexpected thread exits, this code is also removed.